pyomo/contrib/doe/documentation/docs/source/index.rst

```diff
@@ -0,0 +1,453 @@
+############
+Pyomo.DOE
+############
+
+Pyomo.DOE
+===================================
+
+**Pyomo.DOE** (Pyomo Design of Experiments) is a Python library for model-based design of experiments
+using science-based models.
+
+It was developed by **Jialu Wang** and **Alexander W. Dowling** of University of Notre Dame as part of the Carbon Capture Simulation for Industry Impact (CCSI2)
+(https://www.acceleratecarboncapture.org/ ) project, 
+funded through the U.S. DOE Office of Fossil Energy by the Lawrence Berkeley National Laboratory through contract
+DE-AC02-05CH11231 and the National Energy Technology Laboratory, an agency of the United States Government, through a support contract.
+
+If you use Pyomo.DOE, please cite:
+
+[Wang and Dowling, 2022] Jialu Wang, Alexander Dowling (2022), Pyomo.DOE: An Open-Source Package for Model-Based Design of Experiments in Python. (under review)
+
+Methodology Overview
+---------------------
+
+Model-based Design of Experiments (MBDoE) is a technique to maximize the information gain of experiments by directly using
+science-based models with physically meaningful parameters. It is one key component in the model calibration and uncertainty quantification workflow
+shown below:
+
+.. figure:: flowchart.png
+   :scale: 25 %
+   :alt: map to buried treasure
+
+   The exploratory analysis, parameter estimation, uncertainty analysis, and MBDoE are combined into an iterative framework to select, refine, and calibrate science-based mathematical models with quantified uncertainty. The goal of this workflow is to achieve a confident parameter estimation.
+
+Pyomo.DOE provides the exploratory anlaysis and MBDoE capabilities to the Pyomo ecosystem. The user provides one Pyomo model, a set of parameter nominal values,
+the allowable design spaces for design variables, and the assumed observation error model.
+During exploratory analysis, Pyomo.DOE checks if this model parameters can be inferred from the postulated measurements or preliminary data.
+MBDoE then recommends optimized experimental conditions for collecting more data.
+Parameter estimation packages such as **Parmest** can perform parameter estimation using the available data to infer values for parameters,
+and facilitate an uncertainty analysis to approximate the parameter covariance matrix.
+If the parameter uncertainties are sufficiently small, the workflow terminates and returns the final model with quantified parametric uncertainty.
+If not, MBDoE is performed to recommend new experiments to generate new data.
```

Comment:
Replace last sentence: If not, MBDOE recommends optimized experimental conditions to generate new data.